Doc/glossary.rst

```diff
@@ -13,9 +13,16 @@ Glossary
       examples which can be executed interactively in the interpreter.
 
    ``...``
-      The default Python prompt of the interactive shell when entering code for
-      an indented code block or within a pair of matching left and right
-      delimiters (parentheses, square brackets or curly braces).
+      Can refer to:
+
+      * The default Python prompt of the interactive shell when entering code for
+        an indented code block or within a pair of matching left and right
+        delimiters (parentheses, square brackets or curly braces).
+
+      * A object of the :class:`Ellipsis` class.
```

Comment:
I have corrected the sentence in fb07889267. Notice that I cannot use `ellipsis` because the cross reference will not work. Also, notice that the class is used in uppercase [here](https://docs.python.org/3/library/constants.html).

Reply:
Looking here: https://raw.githubusercontent.com/python/cpython/6f0eb93183519024cb360162bdd81b9faec97ba6/Doc/library/stdtypes.rst it looks like :const: seems to have been used before instead of :class:

Reply:
Amended in a7420aa. :+1: